wmn-data.json

```diff
@@ -4329,6 +4329,18 @@
         "m_code" : 404,
         "known" : ["test123", "frickcollection"],
         "cat" : "social",
+        "valid" : true
+       },        
+       {
+        "name" : "pixelfed.social",
+        "uri_check" : "https://pixelfed.social/{account}",
+        "post_body" : "",       
+        "e_code" : 200,
+        "e_string" : "<title>pixelfed</title>",
+        "m_string" : "on pixelfed</title>",

```

Comment:
the title I get when visiting ka bad user account is `<title>pixelfed</title>`. Please check your string.

Reply:
My Bad! I put each one in the wrong string.